lib/taskana-cdi-example/pom.xml

```diff
@@ -19,22 +19,196 @@
     <sonar.tests/>
   </properties>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.wildfly.bom</groupId>
+        <artifactId>wildfly-ee</artifactId>
+        <version>${version.wildfly}</version>
+        <scope>import</scope>
+        <type>pom</type>
+      </dependency>
+      <dependency>
+        <groupId>org.jboss.arquillian</groupId>
+        <artifactId>arquillian-bom</artifactId>
+        <version>${version.arquillian}</version>
+        <scope>import</scope>
+        <type>pom</type>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
   <dependencies>
     <dependency>
-      <groupId>pro.taskana</groupId>
-      <artifactId>taskana-common-logging</artifactId>
```

Comment:
Why is taskana-common-logging removed as dependency?

Reply:
it's a transitive dependency from taskana-cdi. It is not needed it comes automatically.

But I gave you right it is more explicit when you include here. But then we should do this for all modules like `tasana-core`